From f589529278fab95b66fa4e0a33a524fd7dcba9e1 Mon Sep 17 00:00:00 2001
From: Clayton Craft <clayton@craftyguy.net>
Date: Tue, 6 Jan 2026 10:33:17 -0800
Subject: [PATCH 2/2] Revert "arm64: dts: qcom: sc8280xp-el2: Add venus
 video-firmware node"

This reverts commit 2b646cee10691ecab36e411ae092139288358fdc.
---
 arch/arm64/boot/dts/qcom/sc8280xp-el2.dtso | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/sc8280xp-el2.dtso b/arch/arm64/boot/dts/qcom/sc8280xp-el2.dtso
index 71cf230d8bfa6..eb8c22cfabf60 100644
--- a/arch/arm64/boot/dts/qcom/sc8280xp-el2.dtso
+++ b/arch/arm64/boot/dts/qcom/sc8280xp-el2.dtso
@@ -63,9 +63,3 @@ &remoteproc_nsp1 {
 &scm {
 	qcom,shm-bridge-vmid = <QCOM_SCM_VMID_SELF_OWNER>;
 };
-
-&venus {
-	video-firmware {
-		iommus = <&apps_smmu 0x2a02 0x400>;
-	};
-};
-- 
2.52.0

